metrics/collectors/economic/economic.go

```diff
@@ -135,20 +129,28 @@ func (ec *Collector) Collect(ch chan<- prometheus.Metric) {
 	// TODO(samlaf): implement this. probably have to call the BLSOperatorStateRetriever contract?
 	// probably should start using the avsregistry service instead of chainio clients so that we can
 	// swap out backend for a subgraph eventually
-	quorumNums, blsOperatorStateRetrieverOperator, err := ec.avsRegistryReader.GetOperatorsStakeInQuorumsOfOperatorAtCurrentBlock(context.Background(), ec.operatorId)
+	operatorId, err := ec.avsRegistryReader.GetOperatorId(context.Background(), ec.operatorAddr)
 	if err != nil {
-		ec.logger.Error("Failed to get operator stake", "err", err)
+		ec.logger.Error("Failed to get operator id", "err", err)
```

Comment:
shouldn't you return here if there's an error?

Reply:
Not returning, but shouldn't be doing the rest of the logic. Fixed: [9ab28a8](https://github.com/Layr-Labs/eigensdk-go/pull/54/commits/9ab28a8a5753442c03ae3466598c6b1168384328)